domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/ActivityLifecycleObserver.kt

```diff
@@ -0,0 +1,79 @@
+package org.oppia.android.domain.oppialogger.analytics
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+import org.oppia.android.domain.oppialogger.ApplicationStartupListener
+import org.oppia.android.util.system.OppiaClock
+import javax.inject.Inject
+import javax.inject.Singleton
+
+private const val APPLICATION_STARTUP_SCREEN = "application_startup_screen"
+
+/** Observer that observes activity lifecycle. */
```

Comment:
Perhaps clarify that this observer is meant to log analytics events for given activity lifecycle events? The current doc comment is a bit too non-specific.

Reply:
done.